docs/bugzilla

```diff
@@ -21,6 +21,10 @@ branch: you probably want to set this to "master".
 If the central repository option is enabled and there is a "fix",
 "close", or "address" before the bug then that bug is closed.
 
+On bugzilla server side you should disable option `commentonchange_resolution`.
```

Comment:
`If using X option you should disable the `commentonchange_resolution` option in Bugzilla's configuration` or similar